modules/sandboxed-containers-deploying-workloads-with-kata-runtime-web.adoc

```diff
@@ -0,0 +1,77 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-deploying-workloads-with-kata-runtime-web_{context}"]
+= Deploying a workload in a sandboxed container using the web console
+
+{sandboxed-containers-first} installs the kata runtime as a secondary optional runtime on the cluster, and not as the primary runtime.
```

Comment:
Replace kata runtime `kata` as a `runtimeClassName`

Reply:
I'm not sure that's correct. It's installed as a runtime. The `runtimeClassName` field is used to define what the name of the runtime is...